template/package.json

```diff
@@ -30,6 +30,7 @@
         "preact-router": "^3.2.1"
     },
     "devDependencies": {
+        "@teamsupercell/typings-for-css-modules-loader": "^2.2.0",
```

Comment:
why the new dep?

Reply:
As explained, this PR fixes a bug that requires a version update to `css-loader`. The old typings dep, `typings-for-css-modules-loader` doesn't support any newer version of `css-loader`, as it has been unsupported since October of 2017. Therefore it had to be replaced.